### ISSUE TITLE
Hadith Please AND Stories Of Prohephs AND Isnad

### ISSUE THREAD

**Issue by AE-SAY-WAY** [abuibrahim2/quranlookup#32]
Can you include haddith and stories of prophets, and Isnad Please?

**Comment by tashmooSS** [comment]
yes please sahih buhari , riyazussalihin, or kutubi sitte overall. maybe with an option to add our own translation via a pdf or markdown file as well? this would be awesome
